tests/components/zha/test_switch.py

```diff
@@ -564,6 +565,272 @@ async def test_switch_configurable(
     await async_test_rejoin(hass, zigpy_device_tuya, [cluster], (0,))
 
 
+async def test_switch_configurable_custom_on_off_values(
```

Comment:
The following three tests all share quite a bit of code.
Maybe `parametrize` could be used here (easily/in a nice way)? If not, eh.. probably doesn't matter. They are testing (slightly) different things after all, not the same.